### DISCUSSION TITLE
[@types/React-Big-Calendar] EventInteractionArgs is missing resourceId

### DISCUSSION THREAD

**Discussion by amazingBarry** [DefinitelyTyped/DefinitelyTyped#66252] [Issues with a @types package]
I am testing out React Big Calendar. I need the ability to drag and drop to change the resourceId. All of the javascript is there, but EventInteractionArgs does not have resourceId, even though the javascript object includes it.

[a comment by an automated account is not shown]

**Comment by benasher44** [comment]
Fixed in #67810
